xbmc/video/VideoInfoScanner.cpp

```diff
@@ -121,36 +121,45 @@ namespace VIDEO
       // result in unexpected behaviour.
       m_bCanInterrupt = false;
 
-      bool bCancelled = false;
-      while (!bCancelled && !m_pathsToScan.empty())
+
+      if (m_handle)
+        m_handle->SetTitle("Listing media files ...");
```

Comment:
This needs to be a localized string. Look for `g_localizeStrings.Get` usage to get an idea how this works.

Hint: you deleted a line ` m_handle->SetText(g_localizeStrings.Get(20415));` in this PR. Those numbers come from af ile named string.po you can easily find in our code base I guess.